src/main/java/ca/on/oicr/gsi/dimsum/CaseLoader.java

```diff
@@ -697,4 +699,112 @@ private <T> List<T> loadFromJsonArrayFile(FileReader fileReader, ParseFunction<J
+        } else if (PendingState.LIBRARY_PREPARATION.qualifyTest(test)) {
+          builder.incrementLibraryPrepPendingCount();
+        }
+
+        // Library Qualification
+        if (test.getLibraryQualifications().stream().anyMatch(sample -> Boolean.TRUE
+            .equals(sample.getQcPassed())
+            && (sample.getRun() == null || Boolean.TRUE.equals(sample.getDataReviewPassed())))) {
+          builder.incrementLibraryQualCompletedCount();
+        } else if (PendingState.LIBRARY_QUALIFICATION_QC.qualifyTest(test)
+            || PendingState.LIBRARY_QUALIFICATION_DATA_REVIEW.qualifyTest(test)) {
+          builder.incrementLibraryQualPendingQcCount();
+        } else if (PendingState.LIBRARY_QUALIFICATION.qualifyTest(test)) {
+          builder.incrementLibraryQualPendingCount();
+        }
+
+        // Full depth sequncing
+        if (test.getFullDepthSequencings().stream().anyMatch(sample -> Boolean.TRUE
+            .equals(sample.getQcPassed())
+            && (sample.getRun() == null || Boolean.TRUE.equals(sample.getDataReviewPassed())))) {
+          builder.incrementFullDepthSeqCompletedCount();
+        } else if (PendingState.FULL_DEPTH_QC.qualifyTest(test)
+            || PendingState.FULL_DEPTH_DATA_REVIEW.qualifyTest(test)) {
+          builder.incrementFullDepthSeqPendingQcCount();
+        } else if (PendingState.FULL_DEPTH_SEQUENCING.qualifyTest(test)) {
+          builder.incrementFullDepthSeqPendingCount();
+        }
+      }
+      // informatics review
+      if (kase.getRequisition().getInformaticsReviews().stream()
+          .anyMatch(x -> x.isQcPassed())) {
+        builder.informaticsCompletedCount(testSize);
+      }
+      if (PendingState.INFORMATICS_REVIEW.qualifyCase(kase)) {
+        builder.informaticsPendingCount(testSize);
+      }
+
+      // draft report
+      if (kase.getRequisition().getDraftReports().stream()
+          .anyMatch(x -> x.isQcPassed())) {
+        builder.draftReportCompletedCount(testSize);
+      }
+      if (PendingState.DRAFT_REPORT.qualifyCase(kase)) {
+        builder.draftReportPendingCount(testSize);
+      }
+
+      // final report
+      if (kase.getRequisition().getFinalReports().stream()
+          .anyMatch(x -> x.isQcPassed())) {
+        builder.finalReportCompletedCount(testSize);
+      }
+      if (PendingState.FINAL_REPORT.qualifyCase(kase)) {
+        builder.finalReportPendingCount(testSize);
+      }
+
+      // add the counts to each project in the case if the project exists in the
+      // projectSummariesByName
+      for (Project project : kase.getProjects()) {
+        builder.name(project.getName());
+        if (tempprojectSummariesByName.containsKey(project.getName())
+            && !tempprojectSummariesByName.isEmpty()) {
+          tempprojectSummariesByName.get(project.getName()).addCounts(builder);
+        } else {
+          tempprojectSummariesByName.put(project.getName(), builder);
```

Comment:
You'll need to create a new `ProjectSummary.Builder` and use `addCounts` to copy in the counts here. Otherwise, this will happen:

1. Case 1 involves PROJ1 and PROJ2, so in your map, "PROJ1" and "PROJ2" will both be set to the same object
2. Case 2 involves only PROJ1, so you use `addCounts` (above) to copy the case summary into "PROJ1"
3. Since "PROJ2" is the same object as "PROJ1", "PROJ2" now includes the counts for Case 2